src/lexer.h

```diff
@@ -74,10 +75,10 @@ class Lexer
     void deprecation(const char *format, ...);
     void poundLine();
     unsigned decodeUTF();
-    void getDocComment(Token *t, unsigned lineComment);
+    void getDocComment(Token *t, unsigned lineComment, int newParagraph);
```

Comment:
If it can be a bool then it should be.  Most of the cases of using int or unsigned instead are for historical reasons only - either just old-styled code or at some point it had more than two possible values.